frontend/apps/web/src/pages/index.tsx

```diff
@@ -1,152 +1,153 @@
-import { RepositoryData } from '@/mocks'
+import { PageContainer } from '@/containers/PageContainer'
 import {
   Button,
-  Icon,
-  NativeLink,
+  Link,
   PageLayout,
   ProgressBar,
   StackLayout,
   Table,
   Text,
   usePageLayout,
 } from '@filecoin/ui'
-import React, { useState } from 'react'
-import { NavLink, useNavigate } from 'react-router-dom'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { useNavigate } from 'react-router-dom'
+import styled from 'styled-components'
+import { getButton } from './tests'
 
 const Header = props => {
-  const [activeTab, setActiveTab] = useState(1)
-
+  const { t } = useTranslation()
+  const navigate = useNavigate()
   return (
     <PageLayout.Header>
-      <StackLayout>
-        <Text type="heading 5">Systems</Text>
-      </StackLayout>
-      <PageLayout.Tabs>
-        <NavLink to={'/system/overview'}>
-          <PageLayout.Tab
-            onClick={() => setActiveTab(1)}
-            active={activeTab === 1}
+      <HeaderWrapper>
+        <Text type="heading 5">{t('filecoin.systems.systems')}</Text>
+        <Buttons>
+          <Button
+            onClick={() => navigate('/tests')}
+            variant="outline"
+            size="medium"
           >
-            <Icon name="book" size="small" />
-            <Text>Overview</Text>
-          </PageLayout.Tab>
-        </NavLink>
-        <NavLink to={'/system/detailed-view'}>
-          <PageLayout.Tab
-            onClick={() => setActiveTab(2)}
-            active={activeTab === 2}
+            {t('filecoin.allTests.allTests')}
+          </Button>
+          <Button
+            onClick={() => navigate('/behaviors')}
+            variant="outline"
+            size="medium"
           >
-            <Icon name="detailed_view" size="small" />
-            <Text>Detailed view</Text>
-          </PageLayout.Tab>
-        </NavLink>
-      </PageLayout.Tabs>
+            {t('filecoin.allTests.allBehaviours')}
+          </Button>
+        </Buttons>
+      </HeaderWrapper>
     </PageLayout.Header>
   )
 }
 const Home = () => {
+  const {
+    state: { model },
+  } = PageContainer.useContainer()
+
+  const { t } = useTranslation()
+  const systems = model.getAllSystems()
+  const navigate = useNavigate()
   const pageLayout = usePageLayout({
     header: <Header />,
     footer: <PageLayout.Footer />,
   })
-  const navigate = useNavigate()
 
   return (
     <PageLayout {...pageLayout}>
       <PageLayout.Section>
         <Table
-          variant="default"
+          variant="subtle"
           columns={{
-            repository: {
-              header: 'Repository',
+            system: {
+              header: t('filecoin.systems.system'),
+              width: 325,
```

Comment:
At least one column should be responsive, i guess it's best this one to not have the width prop